pkg/generate/app/cmd/newapp.go

```diff
@@ -561,7 +552,17 @@ func (c *AppConfig) RunQuery() (*QueryResult, error) {
 
 func (c *AppConfig) validate() (cmdutil.Environment, cmdutil.Environment, error) {
 	var errs []error
-
+	// Environment File where the parameters are stored
+	if len(c.EnvironmentFile) != 0 {
+		varList, err := app.GetEnvironmentVars(c.EnvironmentFile)
+		if err != nil {
+			return nil, nil, fmt.Errorf("Error reading file %v: %v", c.EnvironmentFile, err)
+		} else {
+			for _, env := range varList {
+				c.Environment = append(c.Environment, env)
```

Comment:
Meaning you can provide both `--env` and `--env-file`, the command will append everything, right? :+1: for this behavior, but if an environment were provided *both* in the env file *and* from `--env` then `--env` must take precedence.